study-frontend/src/Pages/HistoryPage.tsx

```diff
@@ -10,7 +10,9 @@ const HistoryPage = () => {
 
     return (
         <div>
-            <ClosedMenu handleFaBar={handleFaBar}/>
+            <header className='header'>
```

Comment:
Make sure headers, footers and content are in seperate components. You will most likely reuse them throught your entire codebase.